pyproject.toml

```diff
@@ -49,6 +52,7 @@ dependencies    = [
     "orjson==3.9.10",
     "packaging>=23.1",
     "pip>=21.3.1",
+    "python-dateutil==2.8.2",
```

Comment:
We don't use dateutil in our core? This seems/sounds like an integration dependency?